specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/examples/getBillingAccountPolicy.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "billingAccountId": "testBillingAccountId",
+    "api-version": "2021-01-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Subscription/Policies/policyForBillingAccount",
+        "name": "testBillingAccountId",
+        "type": "Microsoft.Subscription/Policies",
```

Comment:
nit: type should be Microsoft.Subscription/policies. resourceType is in lowerCamelCase. Please check other places too.

Reply:
Updated